packages/components/datagrid/src/js/datagrid.html

```diff
@@ -44,14 +48,25 @@
                 <tbody class="oui-datagrid__body"
                     ng-hide="$ctrl.loading">
                     <tr class="oui-datagrid__row oui-action-menu-container"
-                        ng-repeat="row in $ctrl.displayedRows track by $index"
-                        ng-init="rowIndex = $index">
+                        ng-class="{'expandable': $ctrl.expandableRows}"
+                        ng-repeat-start="row in $ctrl.displayedRows track by $index"
+                        ng-init="rowIndex = $index"
+                        ng-click="$ctrl.toggleRowExpansion($index)">
                         <td class="oui-datagrid__cell oui-datagrid__cell-sticky"
                             ng-if="$ctrl.selectableRows">
                             <oui-checkbox model="$ctrl.selectedRows[$index]"
                                             on-change="$ctrl.toggleRowSelection($index, modelValue)">
                             </oui-checkbox>
                         </td>
+                        <td class="oui-datagrid__cell oui-datagrid__cell-sticky"
+                            ng-if="$ctrl.expandableRows">
+                            <button type="button" 
+                                class="button_expand"
```

Comment:
Not BEM compliant with the component classnames.
Since it's an element of the block `oui-datagrid`:

```suggestion
                                class="oui-datagrid__expand-button"
```